about-desktop.css

```diff
@@ -0,0 +1,87 @@
+@import url(./style.css);
+@import url(./desktop.css);
+
+@media screen and (min-width: 768px) {
+  #hero {
+    border: 1px solid var(--gray-color);
+  }
```

Comment:
- [ ] Kindly make sure the background image for the About page matches the original design:
- Your design:
   ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/d2e76fa2-e3d2-4432-a53c-8abdda979858)
- Original design:
  ![image](https://github.com/HtetWaiYan7191/capstone-project/assets/104968236/c72d41fb-2779-4acd-a587-310e812eec84)
- [ ] [OPTIONAL] I highly advise you to keep your stylings in 1 file as separating to multiple files as you did will make it hard to manage by navigating between different files.